### DISCUSSION TITLE
选择文件夹时右键不弹出二级菜单了

### DISCUSSION THREAD

**Discussion by Wenduke6688** [AlistGo/alist#1900] [Q&A]
选择文件夹时右键不弹出二级菜单了![2022-10-06_180004](https://user-images.githubusercontent.com/105553213/194285080-c018cc62-1dc2-4d2a-a9f8-a18430e0d1ac.png)

**Comment by anwen-anyi** [comment]
换个浏览器试试? 或者你的插件?
